modules/mockk-agent-api/src/commonMain/kotlin/io/mockk/proxy/MockKInstantiatior.kt

```diff
@@ -1,5 +1,5 @@
 package io.mockk.proxy
 
 interface MockKInstantiatior {
-    fun <T> instance(cls: Class<T>): T
+    fun <T : Any> instance(cls: Class<T>): T
```

Comment:
The `.kotlin` extension is only defined for `KClass<T : Any>` but from what I've seen it had been `T : Any` in all calling sites anyway.

Reply:
Alright, seems I've missed something. Will revise.